internal/backend/storage/fs/walk.go

```diff
@@ -0,0 +1,59 @@
+package fs
+
+import (
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/gopasspw/gopass/pkg/debug"
+)
+
+func walkSymlinks(path string, walkFn filepath.WalkFunc) error {
+	w := &walker{
+		seen: make(map[string]bool, 100),
```

Comment:
Why 100 and not 400 or 1000?
I feel like 100 might be a bit small, meaning we're going to be doing hundreds of new allocation later on, but 1000 is probably overkill. I'd lean on a 250 or 300 value being more typical, but that's out of my own workflow, so not necessarily representative

Reply:
For an application like gopass I'd argue that this doesn't matter at all. We could also just start at zero. I agree that using a magic constant is not good style. But using a different guessed constant isn't good style either. Maybe I'll just start this at zero.